src/main.c

```diff
@@ -85,7 +90,11 @@ int main(int argc, char *argv[]) {
 		goto error;
 	}
 
+#ifdef _MSC_VER
+    SuspendThread(/*TODO*/);
```

Comment:
This is still WIP, right?

Reply:
Yeah. My main question at the moment is whether running a STUN & TURN server on a node will even make a difference, i.e., if this use-case could be facilitated: https://stackoverflow.com/q/68600741

(my thinking is not which is why I've quasi-abandoned my contribution to the various open-source C STUN/TURN implementations)

Reply:
STUN est basically the standard way to implement NAT hole-punching (in particular in the context of ICE). The STUN server must be outside of the NAT you are trying to traverse (hole-punching always relies on a reachable third node outside the NAT), therefore, deploying a STUN server on one of the two connecting nodes won't make any difference.

Reply:
Yeah that's the same conclusion I came to (eventually).

😢